src/js/components/stock-movement-wizard/AddItemsPage.jsx

```diff
@@ -401,6 +425,10 @@ class AddItemsPage extends Component {
         oldItem.recipient.id : oldItem.recipient;
       const newRecipient = item.recipient && _.isObject(item.recipient) ?
         item.recipient.id : item.recipient;
+      const duplicatedItem = _.find(
```

Comment:
Perhaps we can use findIntex only?

Reply:
In line 463 duplicatedItem.quantityRequested is used. We also need index of this item and that's why we're finding it twice.

Reply:
`lineItemsToBeAdded[duplicatedItemIndex]`